README.md

```diff
@@ -2,58 +2,37 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+One paragraph statement about the project.
```

Comment:
- [ ] Please replace this with the actual statement on the project or remove it completely.